### PR TITLE
export taipy gui base as a proper package

### PR DESCRIPTION
**Pull request by dinhlongviolin1**

### CODE REVIEW

**Review by FabienLelaquais — APPROVED** [review]
🤔

File: frontend/taipy-gui/base/src/exports.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
...
 import { WsAdapter } from "./wsAdapter";
-// import { TaipyApp } from "./app";
+import { ModuleData } from "./dataManager";
 
-export {
-    WsAdapter,
-    // TaipyApp,
-};
+export default TaipyApp;
+export { TaipyApp, createApp, WsAdapter };
```
Comment (FredLL-Avaiga) [inline frontend/taipy-gui/base/src/exports.ts:6]: why export TaipyApp twice ?

**Review by FredLL-Avaiga — APPROVED** [review]
👍

File: frontend/taipy-gui/base/src/exports.ts (hunk 1 above)
Comment (dinhlongviolin1) [reply frontend/taipy-gui/base/src/exports.ts:6]: I want to have a default export in case they want to use it :D

**Pull request merged**